task/pnc-prebuild-git-clone-oci-ta/0.1/patch.yaml

```diff
@@ -0,0 +1,71 @@
+- op: replace
+  path: /metadata/name
+  value: pnc-prebuild-git-clone-oci-ta
+- op: replace
+  path: /metadata/annotations/tekton.dev~1displayName
+  value: pnc prebuild git clone oci trusted artifacts
+- op: replace
+  path: /spec/description
+  value: The pnc-prebuild-git-clone-oci-ta Task will clone a repo from the provided url, apply
+    PNC prebuild modifications from https://github.com/project-ncl/konflux-tooling and store
+    it as a trusted artifact in the provided OCI repository.
+
+- op: add
+  path: /spec/params/-
+  value:
+    name: BUILD_SCRIPT
+    description: The build script to embed with the Containerfile
+    type: string
+- op: add
+  path: /spec/params/-
+  value:
+    name: BUILD_TOOL
+    description: The build tool to use (ant, gradle, maven, sbt).
+    type: string
+- op: add
+  path: /spec/params/-
+  value:
+    name: BUILD_TOOL_VERSION
+    description: The build tool version to use (e.g. 3.9.5)
+    type: string
+- op: add
+  path: /spec/params/-
+  value:
+    name: JAVA_VERSION
+    description: Java version to use (7, 8, 9, 11, 17, 21, 22, 23)
+    type: string
+- op: add
+  path: /spec/params/-
+  value:
+    name: RECIPE_IMAGE
+    description: The image from the build recipe to use
+    type: string
+
+- op: add
+  path: /spec/steps/2
+  value:
+    name: preprocessor
+    image: quay.io/redhat-user-workloads/konflux-jbs-pnc-tenant/konflux-tooling:latest
+    securityContext:
+      runAsUser: 0
+    computeResources:
+      limits:
+        cpu: 300m
+        memory: 512Mi
+      requests:
+        cpu: 10m
+        memory: 512Mi
+    args:
+      - prepare
```

Comment:
nitpick: maybe some comment or extended description would make it more clear what is actually done here.